src/renderer/features/multisig-operations/components/OperationFullInfo.tsx

```diff
@@ -47,10 +46,16 @@ export const OperationFullInfo = memo(({ operation, account }: Props) => {
     chain?.explorers,
   );
 
-  const jsonArgs = useMemo(
-    () => operation.callData && decodeCallData(api, account.accountId, operation.callData, chains),
-    [api, account.accountId, operation.callData, chains],
-  );
+  const jsonArgs = useMemo(() => {
+    if (!operation.callData || !api || !chain) return null;
+
+    try {
+      const extrinsic = getTxFromCallData(api, operation.callData);
```

Comment:
It is strange that we first need to construct extrinsic in order to format call-data. There is nothing in extrinsic that is really required to format a call-data.
So I would suggest to let `formatExtrinsic` actually be `formatCall` and accept `CallBase` instead